### PR TITLE
Deployment Accounts to be send in create service

### PR DESCRIPTION
**Pull request by SauarvD**
### Requirements

* Deployment_Accounts to be sent in the payload of create service

### Description of the Change

* For DEPLOYMENT_ACCOUNTS:
   1. JenkinsFile is modified to send the accounts and mapped regions to environment.oss file.
   2. Mapping of accountIDs to individual regions done in UI.
   3. Payload is modified to include the deployment_accounts as well in create service API. 
   4. Changes in create-serverless-service and in services API to support the newly added tag (DEPLOYMENT_ACCOUNTS)

### Benefits

* Along with other parameters, deployment accounts  details (accountID, region) can be availed in service detail.

### CODE REVIEW

File: core/jazz_create-serverless-service/index.js [hunk 1]
```diff
@@ -234,8 +234,8 @@ var getServiceData = (service_creation_data, authToken, configData) => {
     let primaryAccountCount = 0;
     let deployment_accounts = [];
     // TODO: Make this field required when we support multiple accounts/regions/providers
```
Comment (supritat) [inline core/jazz_create-serverless-service/index.js]: Remove this comment as deployment_accounts field is included in required filed.

File: core/jazz_services/config/prod-config.json [hunk 2]
```diff
@@ -1,8 +1,8 @@
 {
     "services_table": "{inst_stack_prefix}_services_prod",
-    "service_required_fields": ["service", "domain", "type", "created_by", "runtime", "status"],
+    "service_required_fields": ["service", "domain", "type", "created_by", "runtime", "status", "deployment_accounts"],
     "service_update_fields": ["description", "email", "slack_channel", "tags", "repository", "status","metadata","is_public_endpoint"],
-    "service_filter_params": ["service", "domain", "region", "type", "runtime", "created_by", "status"],
+    "service_filter_params": ["service", "domain", "region", "type", "runtime", "created_by", "status", "deployment_accounts"],
```
Comment (supritat) [inline core/jazz_services/config/prod-config.json]: As we can not filter deployment_accounts  its better to remove it from service_filter_params list.

File: core/jazz_services/config/prod-config.json [hunk 3]
```diff
@@ -1,8 +1,8 @@
 {
     "services_table": "{inst_stack_prefix}_services_prod",
-    "service_required_fields": ["service", "domain", "type", "created_by", "runtime", "status"],
+    "service_required_fields": ["service", "domain", "type", "created_by", "runtime", "status", "deployment_accounts"],
```
Comment (supritat) [inline core/jazz_services/config/prod-config.json]: same changes should go in dev-config.json and stg-config.json

File: core/jazz_services/test/test.js [hunk 4]
```diff
@@ -74,6 +74,7 @@ describe('platform_services', function() {
       },
       "body" : {
         "description" : "g0nna_GET_a-L!tt1e_we!rd",
+        "deployment_accounts" : [{"accountId":"12345","region":["us-east-1"],"provider":"aws","primary":true}],
```
Comment (supritat) [inline core/jazz_services/test/test.js]: I think region is not an array

File: core/jazz_create-serverless-service/index.js (hunk 1 above)
Comment (SauarvD) [reply core/jazz_create-serverless-service/index.js]: Removed the comment

File: core/jazz_services/config/prod-config.json (hunk 2 above)
Comment (SauarvD) [reply core/jazz_services/config/prod-config.json]: Removed it from service_filter_params list

File: core/jazz_services/config/prod-config.json (hunk 3 above)
Comment (SauarvD) [reply core/jazz_services/config/prod-config.json]: Added in both the json files

File: core/jazz_services/test/test.js (hunk 4 above)
Comment (SauarvD) [reply core/jazz_services/test/test.js]: Actually region is coming as an array as one account can have multiple regions. So to maintain the uniformity, I made it as an array here in test file.

File: core/jazz_ui/src/app/pages/services-list/services-list.component.ts [hunk 5]
```diff
@@ -385,6 +318,153 @@ export class ServicesListComponent implements OnInit {
...
+    var queryParamKey = 'offset=';
+    var offsetValue = 0;
+    var queryParamValue = offsetValue;
+    $(".pagination.justify-content-center li:nth-child(2)")[0].click();
+    // this.pageSelected = 1;
+    
+    this.addQueryParam(queryParamKey, queryParamValue, false);
+    
+    if (event.key == col.key) {
+    queryParamKey = col.key + '=';
```
Comment (supritat) [inline core/jazz_ui/src/app/pages/services-list/services-list.component.ts]: Fix the indentation of your code changes.

**Pull request merged**